submissions/yanyshpolska/OOP-exercise/index.js

```diff
@@ -0,0 +1,78 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Yanyshpolska/a-tiny-JS-world
+   Web app: https://yanyshpolska.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+  constructor(name, saying, sex) {
+    this.name = name;
+    this.saying = saying;
+    this.sex = sex;
+  }
+  getInfo() {
+    return this.name + ", " + this.saying + ", " + this.sex;
+  }
+}
+
+class Human extends Inhabitant {
+  hands = 2;
+  legs = 2;
+  constructor(...args) {
+    super(...args);
+  }
+  getInfo() {
+    return (
+      super.getInfo() +
+      ", " +
+      this.hands +
+      ", " +
+      this.legs +
+      ", " +
+      this.constructor.name
```

Comment:
Benefit from `Array#join`.
Please explain benefits.

Reply:
I have tried the solution with the Array.join and it worked for me. I found a more effortless and more elegant solution than concatenating strings manually. However in the Array.join I can't control the order of parameters so I need to arrange them correctly in the class itself.
For this particular case, it works fine, so I switched to Array.join.
Thanks for the suggestion.

Reply:
I didn't look into fixes, but this part "I can't control the order of parameters so I need to arrange them correctly in the class itself." sounds confusing.
What stops you from arranging any items before joining them?
E.g. `[ prop9, prop1, prop3].join("")`

Reply:
Took a peek into the changes. `Object` methods are still a no go. The reasons behind remain the same since pre-OOP TJSW app.

Reply:
My bad. Totally forgot than Object methods are not recommended to be used.